app/src/main/java/io/gnosis/safe/ui/transaction/list/TransactionPagingRepository.kt

```diff
@@ -0,0 +1,26 @@
+package io.gnosis.safe.ui.transaction.list
+
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import io.gnosis.data.models.Transaction
+import io.gnosis.data.repositories.SafeRepository
+import io.gnosis.data.repositories.TransactionRepository
+import kotlinx.coroutines.flow.Flow
+import pm.gnosis.model.Solidity
+
+class TransactionPagingRepository(
```

Comment:
`Repository` is a big word in our app :D Maybe calling it `TransactionPagingHelper` would be a bit more accurate?

Reply:
repository suits here better. This class provides pages of data, so it is a repository in the sense of a word. Helper is more misleading here as it usually indicates some utilitarian functionality

Reply:
A repository, means that data could potentially be stored. This class, doesn't seem to be doing that. It orchestrates dependencies and maps them into a different wrapper type, which sounds to me like the sort of stuff a helper class does.

Reply:
it can potentially store pages. We could use [RemoteMediator](https://developer.android.com/reference/kotlin/androidx/paging/RemoteMediator) and store pages in a database. So this page provider can be extended to store data if we decide to support offline tx browsing. Repository in the name is a good reminder of this potential

Reply:
I don't think it is a good practice to name classes after what they potentially could become. If you feel like this is an improvement we would need in the future, I would say that placing the link you provided in a comment within this class is a much better way to remind ourselves while at the same time naming the class for what it actually is at the time of the implementation.